frontend/src/components/widgets/DateInput/DateInput.tsx

```diff
@@ -231,15 +253,20 @@ class DateInput extends React.PureComponent<Props, State> {
             CalendarContainer: {
               style: {
                 fontSize: fontSizes.sm,
+                paddingRight: theme.spacing.sm,
+                paddingLeft: theme.spacing.sm,
+                paddingBottom: theme.spacing.sm,
+                paddingTop: theme.spacing.sm,
               },
             },
             Week: {
               style: {
-                fontSize: fontSizes.sm,
+                fontSize: fontSizes.twoSm,
```

Comment:
I think you can change this back to `sm`, this will bring it closer to the current look.

Reply:
Thanks 👍 Went back 👍